test/helpers/GameFlowWrapper.js

```diff
@@ -50,7 +48,7 @@ class GameFlowWrapper {
             return player.hasPrompt('Waiting for opponent to take an action or pass') ? 1 : 0;
         };
 
-        var playersInPromptedOrder = this.allPlayers.sort((playerA, playerB) =>
+        var playersInPromptedOrder = this.allPlayers.concat().sort((playerA, playerB) =>
```

Comment:
Good catch on needing the copies. Are there any alternative shorthands for doing it besides `concat()` that might be a little clearer? The `concat()` by itself is pretty unintuitive for folks who don't know about the behavior, if I saw this in the code previously I probably would've assumed it was a bug. Not a hard block, just curious if there are easy alternatives

Reply:
alternative is [...this.allPlayers] to destructure the array into another array.